src/TourCMS.php

```diff
@@ -40,6 +40,8 @@ class TourCMS {
 	protected $result_type = "";
 	protected $timeout = 0;
 	protected $last_response_headers = array();
+    protected $user_agent = "";
+    protected $prepend_caller_to_user_agent = true;
```

Comment:
Small thing, spacing.
```suggestion
	protected $last_response_headers = array();
	protected $user_agent = "";
	protected $prepend_caller_to_user_agent = true;
```